docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -21,9 +22,7 @@ activate AddressBookParser
 
 create DeleteCommandParser
 AddressBookParser -> DeleteCommandParser
-activate DeleteCommandParser
 
-DeleteCommandParser --> AddressBookParser
```

Comment:
This shouldn't be removed, it's the constructor for DeleteCommandParser

Reply:
fixed in latest commit
https://github.com/AY2021S1-CS2103T-T12-1/tp/pull/169/commits/c276c055ccc67713af31fe5506ca399aa0cee012